### PR TITLE
Enable image mirroring for kubefed artifacts

### PR DESCRIPTION
**Pull request by abhat**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -294,7 +294,7 @@ image-mirror-setup:
 	$(MAKE) apply WHAT=cluster/ci/jobs/image-mirror.yaml
 .PHONY: image-mirror-setup
 
-knative-image-mirror-setup:
+knative-image-miror-setup:
```
Comment (smarterclayton) [inline Makefile]: typo?

Comment (abhat) [reply Makefile]: more like fat-fingering a delete key somewhere inadvertently. let me fix it.

**Pull request merged**